(PC-30566)[API] fix: doc: improve/correct stuff (#12947)
pass-culture/pass-culture-main: api/documentation/docs/tutorials/create-a-collective-offer.md

## Setup

First, set up your provider if you haven't already done so (more information [**here**](/docs/mandatory-steps/request-a-provider-account#how-to-get-a-provider-account).

Next step is to ensure that your API key is linked to at least one venue.

:::warning

> Note that in an integration environment, all venues are linked to an Adage ID, which mean you are allowed to created a collective offer, in order to test it. However, in the production environment it won’t be the case : a venue is not automatically referenced Adage. Please ensure that your linked venues can be used for collective offers.

Reviewer: ```suggestion
Note that in an integration environment, all venues are linked to an Adage ID, which means you are allowed to created a collective offer, in order to test it. However, in the production environment it won’t be the case : a venue is not automatically referenced Adage. Please ensure the venues for which you are planning to create collective offers in production are linked to and Adage ID.
```
